NEWS.md

```diff
@@ -47,6 +66,12 @@ The feed template for the URLHaus feed contained a spelling error:
 The correct name for the parameter "delimeter" is "delimiter". Please fix your configured bots.
 The `intelmqctl upgrade-config` command automatically fixes a configuration if the misspelling is detected.
 
+#### IntelMQ-Manager -> IntelMQ-API
+In `2.3.0` release `IntelMQ-Manager` is getting replaced by `IntelMQ-API`, only frontend files from `IntelMQ-Manager` are used. So you have to reinstall the `IntelMQ-Manager` & install the `IntelMQ-API`, in order to update your local installation. Follow the instructions in our [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html) to install `IntelMQ-API` & setup authentication. Unfortunately you can't keep current authentication users, you have to add users as described in the [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html#adding-a-user).
```

Comment:
This is part of the news entry for the 2.3.0 release, I don't think its necessary to repeat the version here. `only frontend files from IntelMQ-Manager` are used should probable read: "The IntelMQ-Manager release only contains the files for the web frontend". You should not have to reinstall anything, the upgrade path should handle the move to intelmq-api (thats why intelmq-manager depends on intelmq-api).